### ISSUE TITLE
Syntax error

### ISSUE THREAD

**Issue by fernandoblalves** [tweepy/tweepy#1088]
Hi,

Running pip 18.0, tweepy 3.6.0, python 3.7

On importing tweepy modules
```
from tweepy import TweepError
from tweepy.streaming import StreamListener
from tweepy import Stream
```

I'm getting the following error:

```
Traceback (most recent call last):
  File "/home/fernando/PycharmProjects/get_tweets/get_tweets.py", line 6, in <module>
    from tweepy import TweepError
  File "/usr/lib/python3.7/site-packages/tweepy/__init__.py", line 17, in <module>
    from tweepy.streaming import Stream, StreamListener
  File "/usr/lib/python3.7/site-packages/tweepy/streaming.py", line 358
    def _start(self, async):
                         ^
SyntaxError: invalid syntax
```

Am I doing something wrong?

Thanks

**Comment by Harmon758** [comment]
Duplicate of #1017 and fixed with #1042.
When submitting issues, please make sure you're on the most recent development version.
Otherwise, see #1063.

**Comment by fernandoblalves** [comment]
Fixed using
```
pip uninstall tweepy
pip install git+https://github.com/tweepy/tweepy
```
